src/main/app/windowManager.js

```diff
@@ -84,6 +84,12 @@ class WindowManager extends EventEmitter {
    */
   add (window) {
     this._windows.set(window.id, window)
+
+    if (!this._appMenu.has(window.id)) {
+      // TODO: Build a default menu for macOS.
```

Comment:
Window and Linux do not need default menu?

Reply:
No, it's not needed.